quantecon/game_theory/normal_form_game.py

```diff
@@ -276,8 +276,7 @@ def best_response(self, opponents_actions, tie_breaking='smallest',
 
         Parameters
         ----------
-        opponents_actions : array_like(int or array_like(float)) or
-                            array_like(int, ndim=1) or scalar(int)
+        opponents_actions : scalar(int) or array_like
```

Comment:
If we ignore the maximum length 72, it would have been
```py
        opponents_actions : scalar(int) or array_like(int, float, or array_like(foat))
```

See https://github.com/QuantEcon/QuantEcon.py/issues/287#issuecomment-286448104.